packages/simcore-sdk/src/simcore_sdk/node_ports_common/exceptions.py

```diff
@@ -123,3 +123,11 @@ class NodeNotFound(NodeportsException):
 
     def __init__(self, node_uuid):
         super().__init__(f"the node id {node_uuid} was not found")
+
+
+class SymlinkToSymlinkNotSupportedException(NodeportsException):
+    """Not possible to upload a symlink to a symlink"""

```

Comment:
upload? Is this only used for upload? then maybe add this to the name of the exception.
Also you might want to add what symlink is faulty.. it would ease for debugging